### ISSUE TITLE
Updating to 2022.6.2 breaks nginx config

### ISSUE THREAD

**Issue by Berzerker** [bitwarden/self-host#43]
After updating to 2022.6.2, I was unable to access the web interface for my self-hosted instance. I checked the nginx error.log and saw the following error flooding the log:

`[emerg] 1#1: invalid number of arguments in "server_name" directive in /etc/nginx/conf.d/default.conf:11`

Checking `bwdata/nginx/default.conf` line 11, I saw the `server_name` directive was blank. Adding in localhost and restarting bitwarden solved the issue.

I did not see any errors during the update and made no other changes when updating.

**Comment by Berzerker** [comment]
Want to bump this back up, almost a year to the day this happened for a second time when updating to 2023.7

**Comment by Berzerker** [comment]
Bumping this *again* to say that it's still not fixed for me in 2024.1.0

**Comment by Berzerker** [comment]
And one again...2024.6.1 still experiences this issue.

**Comment by Berzerker** [comment]
2.5 years later this is still an issue when updating.

**Comment by vinnyperella** [comment]
Are you using the unified beta? I'm using the regular non-beta version and I've never run in to this issue while upgrading. Currently on the latest version.

**Comment by Berzerker** [comment]
> Are you using the unified beta? I'm using the regular non-beta version and I've never run in to this issue while upgrading. Currently on the latest version.

I don't believe so, I don't think I've ever set it up and none of my docker containers have any beta tags.

**Comment by vinnyperella** [comment]
> Well not sure what fixed it, but I ran an `updateconf` again and it seems to have done the trick, it's all working as expected now.

Glad it's working perhaps it couldn't parse, or parsed in a strange way, the multiple URL's you had set and left it blank. I think if you modify the config.yml you have to run "./bitwarden.sh rebuild" unless you tried that.
